data/sql/updates/pending_db_world/rev_1671645973738924500.sql

```diff
@@ -0,0 +1,7 @@
+--
+-- Recipe: Super Mana Potion 30m
+UPDATE `npc_vendor` SET `incrtime`=1800 WHERE `entry`=18005 AND `item`=22907 AND `ExtendedCost`=0;
+UPDATE `npc_vendor` SET `incrtime`=1800 WHERE `entry`=19837 AND `item`=22907 AND `ExtendedCost`=0;
+-- Recipe: Elixir of Major Frost Power 10m
+UPDATE `npc_vendor` SET `incrtime`=600 WHERE `entry`=18005 AND `item`=22902 AND `ExtendedCost`=0;
+UPDATE `npc_vendor` SET `incrtime`=600 WHERE `entry`=19837 AND `item`=22902 AND `ExtendedCost`=0;
```

Comment:
```suggestion
UPDATE `npc_vendor` SET `incrtime`=1800 WHERE `entry` IN (18005, 19837) AND `item`=22907 AND `ExtendedCost`=0;
-- Recipe: Elixir of Major Frost Power 10m
UPDATE `npc_vendor` SET `incrtime`=600 WHERE `entry` IN (18005, 19837) AND `item`=22902 AND `ExtendedCost`=0;
```

Also, it doesn't look like 19837 has item 22902
![image](https://user-images.githubusercontent.com/100873791/209013526-8a329f1d-223d-4cdd-a813-8fbb94814f06.png)